src/main/webapp/components/search-form-select/search-form-select.stories.tsx

```diff
@@ -1,11 +1,45 @@
 import React from 'react'
 import { storiesOf } from '../../@storybook/react'
+import { action } from '@connexta/ace/@storybook/addon-actions'
 const stories = storiesOf('Search Form Select', module)
-import SearchFormSelect from './search-form-select'
+import { SearchFormSelect } from './search-form-select'
+import { ApolloError } from 'apollo-client'
 stories.add('Basic', () => {
+  const searchForms = [
+    { title: 'Search Form 1' },
+    { title: 'Search Form 2' },
+    { title: 'Another Search Form' },
+    { title: 'Yet Another' },
+  ]
   return (
     <div style={{ width: '100vw', height: '100vh' }}>
-      <SearchFormSelect />
+      <SearchFormSelect
+        searchForms={searchForms}
+        onSelect={action('onSelect')}
```

Comment:
since onSelect is of the type (queryType) => void, can you do something along the lines of `query => action('onSelect')(query.title)` to log what the form was switched to?

Reply:
as is, it logs the entire object:
![image](https://user-images.githubusercontent.com/10997562/79491767-2d6ef300-7fdc-11ea-9d0a-970533b80594.png)

Reply:
oh, nice! that's good then